style.css

```diff
@@ -20,3 +20,20 @@ Underscores is distributed under the terms of the GNU GPL v2 or later.
 Normalizing styles have been helped along thanks to the fine work of
 Nicolas Gallagher and Jonathan Neal http://necolas.github.com/normalize.css/
 */
+
+/*--------------------------------------------------------------
+>>> TABLE OF CONTENTS:
+----------------------------------------------------------------
+# Generic
+	- Normalize
+
+--------------------------------------------------------------*/
+
+/*--------------------------------------------------------------
+# Generic
+--------------------------------------------------------------*/
+
+/* Normalize
+--------------------------------------------- */
+
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
@devanshijoshi9 @robicse11127 Where are we using this `normalize.css`?

Reply:
@RahiDroid I don't know the exact usage of normalis.css in this theme but in [_normalize.scss](https://github.com/rtCamp/blank-theme/blob/master/assets/src/sass/2-generic/_normalize.scss) file I guess we have used some normalise.css classes. So I added the Normalize comment here.

Reply:
@devanshijoshi9 Ah, please remove it from this file then.

Reply:
Done 👍

Reply:
```
/*--------------------------------------------------------------
>>> TABLE OF CONTENTS:
----------------------------------------------------------------
# Generic
	- Normalize
--------------------------------------------------------------*/

/*--------------------------------------------------------------
# Generic
--------------------------------------------------------------*/
```
Remove these as well, please.